health/REFERENCE.md

```diff
@@ -394,21 +394,21 @@ By defining labels inside of `netdata.conf`, you can now apply labels to alarms.
 line to any alarms you'd like to apply to hosts that have the label `room = server`.
 
 ```yaml
-label: room = server
+host lables: room = server
 ```
 
 You can also combine labels when applying them to alarms. For example, if you want to raise a specific alarm only for hosts 
 inside a room that were installed at a specific time, you can write the following label line:
 
 ```yaml
-label: room = workstation AND installed = 201705
+host lables: room = workstation AND installed = 201705
```

Comment:
Simple patterns don't support the `AND` operator.

lables -> labels.

Reply:
I used `:% s/label:/host lables:/`, so the error was replicated, thank you very much!